charts/alfresco-search-service/templates/_helpers.tpl

```diff
@@ -31,7 +42,11 @@ Get Alfresco Search container Port ("internal")
 Get Alfresco Search Service Port
 */}}
 {{- define "alfresco-search.svcPort" -}}
-{{- .Values.service.externallPort | default 80 | int -}}
+{{- $svcPort := 80 }}
+{{- if hasKey .Values "service" }}
+  {{- $svcPort := default 80 .Values.service.externalPort }}
```

Comment:
no need to default, it's already 80

Reply:
well I still want to cover for cases where `.Values.service.externalPort` is not set, but I guess you'll prefer this other way of doing things